packages/react/src/components/Button/Button.test.tsx

```diff
@@ -619,18 +581,18 @@ describe('Button', () => {
           }}
         />,
       );
-      const primaryButtonDOM: HTMLButtonElement = button.getElementsByTagName('button')[0] as HTMLButtonElement;
+      const primaryButton = getAllByRole('button')[1];
 
       // in a normal scenario, when we do a touchstart we would also cause a
       // click event to fire. This doesn't happen in the simulator so we're
       // manually adding this in.
-      ReactTestUtils.Simulate.touchStart(primaryButtonDOM);
-      ReactTestUtils.Simulate.click(primaryButtonDOM);
-      expect(button.getAttribute('aria-expanded')).toEqual('true');
+      fireEvent.touchStart(primaryButton);
+      fireEvent.click(primaryButton);
```

Comment:
In this case, I needed to use `fireEvent` to trigger a `touchstart` since `userEvent` does not have a touch method. Without a `touchstart` event, the primary button does not open the menu. I've gone ahead and replaced `fireEvent.click` with `userEvent.click` though